tests/ti.analytics.addontest.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* global Ti */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions');
+
+describe('Titanium.Analytics', function () {
+	it.androidMissing('.optedOut', function () {
```

Comment:
@garymathews @ypbnv Once Android is merged, we can update this one to work or all platforms.